src/core/utils.js

```diff
@@ -102,3 +106,10 @@ export function loadAddonIfNeeded(
   return fetchAddon({ slug, api: state.api })
     .then(({ entities }) => dispatch(loadEntities(entities)));
 }
+
+export function getAction(type) {
```

Comment:
I feel like this should be in `core/tracking`. It might make sense to specify that this is a tracking action as well by naming it `trackingAction` or `getTrackingAction` since "action" is a term used in redux as well.